docusaurus.config.js

```diff
@@ -24,6 +24,12 @@ const config = {
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'warn',
 
+  // Used for creating diagrams
+  markdown: {
+    mermaid: true,
+  },
+  themes: ['@docusaurus/theme-mermaid'],
+
   // Even if you don't use internalization, you can use this field to set useful
   // metadata like html lang. For example, if your site is Chinese, you may want
```

Comment:
Refactor meta-data (used in other places in this documentation) to metadata like you have below.  lang. should be spelled out. 
```suggestion
  // Even if you don't use internalization, you can use this field to set useful
  // metadata like html language. For example, if your site is Chinese, you may want
```